src/components/FriendList/FriendsNotification.js

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import Typography from "@mui/material/Typography";
+import {StudyRoomCard} from "../StudyRoom/CommonResources";
+import {Stack} from "@mui/material";
+import Button from "@mui/material/Button";
+import ClearIcon from "@mui/icons-material/Clear";
+import CheckIcon from '@mui/icons-material/Check';
+import axios from "axios";
+
+
+export default function FriendNotification() {
+    const [requestSent, setRequestSent] = React.useState([]);
+    const [requestReceived, setRequestReceived] = React.useState([]);
+    const ownerEmail = JSON.parse(localStorage.getItem("email"));
+
+    function handleCancel(index){
+        const id = requestSent[index]._id;
+        axios.post(`${process.env.REACT_APP_BASE_URL}friend/cancel-request`,{requestId:id})
+            .then(() => {
+            })
+            .catch(err => {console.log('Error:', err)});
+         window.location.reload();
+    }
+
+    function handleReject(index){
+        const id = requestReceived[index]._id;
+        axios.post(`${process.env.REACT_APP_BASE_URL}friend/answerFriendRequest`, {answer:"declined", requestId:id})
+            .then(() => {
+            })
+            .catch(err => {console.log('Error:', err)});
+        window.location.reload();
+    }
+
+    function handleAccept(index){
+        const id = requestReceived[index]._id;
+        axios.post(`${process.env.REACT_APP_BASE_URL}friend/answerFriendRequest`, {answer:"accepted", requestId:id})
+            .then(() => {
+            })
+            .catch(err => {console.log('Error:', err)});
+        window.location.reload();
+    }
+
+    React.useEffect( ()=>{
+        //friend request received
+        axios.get(`${process.env.REACT_APP_BASE_URL}friend/incoming-requests/${ownerEmail}`)
+            .then(res => {
+                setRequestReceived(res.data);
+            })
+        // //friend request sent
+        axios.get(`${process.env.REACT_APP_BASE_URL}friend/outgoing-requests/${ownerEmail}`)
+            .then(res => {
+                setRequestSent(res.data);
+            })
+    },[])
+
+    const friendNotification = (
```

Comment:
you can return here directly if you are not using friendNotification anywhere else

Reply:
it's used somewhere else, in the friend list home page